models/media.py

```diff
@@ -81,3 +82,29 @@ def cdphotothread_thread_url(self):
     @property
     def youtube_url(self):
         return 'https://www.youtube.com/embed/{}'.format(self.foreign_key)
+
+    @property
+    def imgur_url(self):
+        return 'https://imgur.com/{}'.format(self.foreign_key)
+
+    @property
+    def imgur_direct_url(self):
+        return 'https://i.imgur.com/{}h.jpg'.format(self.foreign_key)
```

Comment:
Again, see above? We can standardize on an extension to use, because just submitting https://imgur.com/rdwAFcH doesn't provide any info about the filetype